src/projects.js

```diff
@@ -0,0 +1,202 @@
+/*
+Get Subjects For Projects
+This script pulls every Subject from a list of Projects, then creates a new
+Datasette-ready database for that project.
+
+Inputs/configurables:
+- (hardcoded) PROJECTS: list of target projects
+- (hardcoded) OUTPUT_DIR: Output directory
+- (hardcoded) TABLE_PREFIX: Prefix for database table name
+
+Output:
+- A Datasette-ready database for each project, named project-{project_id}
+  (e.g. project-1234)
+
+Notes:
+- Currently used by 2023 Community & Crowds's Community Catalog
+  (https://github.com/zooniverse/community-catalog)
+*/
+
+const fs = require('fs')
+const { unparse } = require('papaparse')
+
+/*
+Input/Configurables
+--------------------------------------------------------------------------------
+ */
+const OUTPUT_DIR = './data/projects/'  // Trailing slash
+const TABLE_PREFIX = 'proj_'
+
+// Note: this can be separated into its own JSON file
+const PROJECTS = [
+  {
+    "name": "Community Catalog (Stable Test Project)",
+    "id": 21084,
+    "metadata_fields": [ "Item", "Notes", "folder", "image1", "image2", "#Hazard", "Oversize", "group_id", "Condition", "internal_id", "part_number", "Photographer", "#Other Number", "picture_agency", "Sensitive_Image", "Problematic_Language", "Notes on Problematic Language" ]
+  }
+]
+/*
+--------------------------------------------------------------------------------
+ */
+
+const headers = {
+  'Content-Type': 'application/json',
+  Accept: 'application/vnd.api+json; version=1'
+}
+
+async function main () {
```

Comment:
```suggestion
async function main() {
```

Reply:
The interpreter allows whitespace in function declarations, but it looks odd. I’m not sure about the readability of these function declarations. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function